ops/solana/solana.go

```diff
@@ -180,21 +220,80 @@ func (d Deployer) TransferLINK() error {
 
 func (d Deployer) InitOCR(keys []map[string]string) error {
 
-	jsonKeys, err := json.Marshal(keys)
+	fmt.Println("Setting up OCR Feed:")
+
+	fmt.Println("Begin set config...")
+	err := d.gauntlet.ExecCommand(
+		"ocr2:begin_offchain_config",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("version", "1"),
+		d.gauntlet.Flag("state", d.States[OCRFeed]),
+	)
+	if err != nil {
+		return errors.New("begin OCR 2 set config failed")
+	}
+
+	S := []int{}
+	offChainPublicKeys := []string{}
+	peerIDs := []string{}
+	for _, k := range keys {
+		S = append(S, 1)
+		offChainPublicKeys = append(offChainPublicKeys, k["OCROffchainPublicKey"])
+		peerIDs = append(peerIDs, k["P2PID"])
+	}
+
+	input := map[string]interface{}{
+		"deltaProgressNanoseconds": 2 * time.Second,
```

Comment:
We should have this object properly typed and not just use `map[string]interface{}`. Good for now to move quicker, but aadd a `// TODO` for later please.